api/app/tests/apps/signals/test_api_v1_endpoints.py

```diff
@@ -1080,6 +1080,70 @@ def is_same_location(a, b):
             # Check category assignment
             self.assertIsNotNone(child_json['category'])
 
+    def test_split_children_must_inherit_parent_images(self):
+        # Split the signal, take not of the returned children
+
+        def md5(fname):
```

Comment:
consider: `md5` -> `calculate_md5` (I have a preference for verbs for function names.)

Also factor out this helper function, it is repeated below.